pinot-minion/src/main/java/com/linkedin/pinot/minion/executor/BaseMultipleSegmentsConversionExecutor.java

```diff
@@ -80,6 +80,7 @@
       for (int i = 0; i < downloadURLs.length; i++) {
```

Comment:
for (String downloadURL : downloadURLs) {
    ...
}

Reply:
I like that more too, but the i is used to name the tarredSegmentFile

Reply:
Oh, missed that, my bad